index.bs

```diff
@@ -111,32 +172,71 @@ dictionary MediaStreamTrackProcessorInit {
 1. If |init|.{{MediaStreamTrackProcessorInit/track}} is not a valid {{MediaStreamTrack}},
     throw a {{TypeError}}.
 2. Let |processor| be a new {{MediaStreamTrackProcessor}} object.
-3. Assign |init|.{{MediaStreamTrackProcessorInit/track}} to the `[[track]]`
-    internal slot of |processor|.
-3. Assign |init|.{{MediaStreamTrackProcessorInit/maxBufferSize}} to the `[[maxBufferSize]]`
-    internal slot of |processor|.
-4. Return |processor|.
+3. Assign |init|.{{MediaStreamTrackProcessorInit/track}} to |processor|.`[[track]]`.
+4. Assign |init|.{{MediaStreamTrackProcessorInit/maxBufferSize}} to |processor|.`[[maxBufferSize]]`.
```

Comment:
Should be appMaxBufferSize? Is maxBufferSize initialized to the same value?

Reply:
It should be appMaxBufferSize. I initialized both, but maxBufferSize can be changed by the UA at any time.

Reply:
Yes, it should be appMuxBufferSize. Updated the text to initialize maxBufferSize to the same value, but it can be changed at any time by the UA.